### DISCUSSION TITLE
Neutralino + Capacitor?

### DISCUSSION THREAD

**Discussion by bitnom** [neutralinojs/neutralinojs#734] [Ideas]
Would it be possible to have a neutralino template for [Capacitor](https://capacitorjs.com/)? I'm just starting out, wanting to use neutralino for a project but I'm looking to see what if anything I'll need to change when building with capacitor. Obviously calls to neutralino API will need to be adapted. I'm thinking in terms of `Neutralino.init`

**Comment by brody2consult** [comment]
That sounds similar to using Capacitor with Electron.

**Comment by bitnom** [comment]
> That sounds similar to using Capacitor with Electron.

It's just that if you think about it, it seems much simpler to port a neutralino project to capacitor than the other way around.

**Comment by brody2consult** [comment]
That makes sense to me. I do think it would be really nice to see this happen:)
